argmin/src/solver/linesearch/mod.rs

```diff
@@ -55,16 +55,16 @@ pub use self::morethuente::MoreThuenteLineSearch;
 /// ```
 /// use argmin::solver::linesearch::LineSearch;
 ///
-/// struct MyLineSearch<P, F> {
-///     // `P` is the type of the search direction, typically the same as the parameter vector
-///     search_direction: P,
+/// struct MyLineSearch<D, F> {
+///     // `D` is the type of the search direction, typically the same as the parameter vector
```

Comment:
Thanks for also fixing the docs. There is only one small suggestions:
```suggestion
///     // `D` is the type of the search direction
```

Reply:
Should be resolved now. I also squashed all the fixup commits.